src/Plugins/SimplnxCore/src/SimplnxCore/Filters/Algorithms/ComputeBoundingBoxStats.cpp

```diff
@@ -111,6 +111,10 @@ class ComputeBaseStatsImpl
   , m_UnifiedBounds(unifiedBounds)
   , m_StatsVector(statsVector)
   {
+    if constexpr(std::is_same_v<T, bool>)
```

Comment:
I think we should use a modified version of `ExecuteDataFunction` so that `bool` doesn't get used and compiled. And also check that the filter disallows `bool`.